ui/public/i18n/language.json

```
@@ -27,32 +27,19 @@
     "roundcubemail_fqdn": "Roundcube hostname (FQDN)",
     "lets_encrypt": "Request LE Certificate",
     "http_to_https": "Force HTTPS",
-    "mail_server_fqdn": "Mail server hostname (FQDN)",
-    "encrypt_imap": "IMAP encryption",
-    "none": "None",
-    "starttls": "STARTTLS",
-    "tls": "TLS",
-    "imap_port": "IMAP TCP port",
-    "encrypt_smtp": "SMTP encryption",
-    "smtp_port": "SMTP TCP port",
-    "tls_verify_smtp": "Use a valid certificate for SMTP",
-    "tls_verify_imap": "Use a valid certificate for IMAP",
+    "mail_server_fqdn": "Mail server hostname",
     "enabled": "Enabled",
     "disabled": "Disabled",
     "advanced": "Advanced",
     "plugins": "Plugins",
     "upload_max_filesize": "Maximum size for attachments (MB)",
     "upload_max_filesize_invalid_type": "Must be an integer",
-    "smtp_port_invalid_type": "Must be a valid TCP port",
-    "imap_port_invalid_type": "Must be a valid TCP port",
-    "test_imap": "IMAP error",
-    "test_smtp": "SMTP error",
-    "cannot_connect_to_server": "Cannot connect to server",
-    "valid_certificate_verification_error": "The certificate is not valid for the hostname or self signed",
     "configuring": "Configuring",
     "instance_configuration": "Configure Roundcube",
     "placeholder_plugins": "Plugins list (comma separated: plugin1,plugin2,plugin3)",
-    "choose": "Choose encryption"
+    "choose_mail_server": "Select a mail server as endpoint",
+    "choose_the_mail_server_to_use": "Roundcube needs to connect to a mail server, select one in the list",
+    "mail_server_is_not_valid": "This mail server cannot be used by roundcube webmail"
```

```suggestion
    "mail_server_is_not_valid": "This mail server cannot be used by Roundcube webmail"
```
What is the condition that raises this error? Why a mail server in the list valid should be invalid?